dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/Teradata14LogsConnector.java

```diff
@@ -71,15 +84,84 @@ public Teradata14LogsConnector() {
         super("teradata14-logs");
     }
 
-    private static class LSqlQueryFactory extends TeradataLogsConnector.TeradataLogsJdbcTask {
+    private static abstract class TeradataLogsJdbcTask extends AbstractJdbcTask<Void> {
```

Comment:
I would suggest to rename TeradataLogsJdbcTask to Teradata14LogsJdbcTask in order for this name not to clash with the other TeradataLogsJdbcTask class (PR #158).